internal/secretshape/testfixture/private_key.go

```
@@ -0,0 +1,115 @@
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/ed25519"
+	"crypto/elliptic"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/pem"
+	"math/big"
+	"time"
+
+	"golang.org/x/crypto/openpgp/packet"
```

golangci-lint (2.12.2)

[error] SA1019: "golang.org/x/crypto/openpgp/packet" is deprecated: this package is unmaintained except for security fixes. New applications should consider a more focused, modern alternative to OpenPGP for their specific task. If you are required to interoperate with OpenPGP systems and need a maintained package, consider a community fork. See https://golang.org/issue/44226. 

(staticcheck)